SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -99,15 +99,24 @@ struct FConnectionConfig
 		if (LinkProtocolString.Compare(TEXT("Tcp"), ESearchCase::IgnoreCase) == 0)
 		{
 			LinkProtocol = WORKER_NETWORK_CONNECTION_TYPE_MODULAR_TCP;
+			return;
 		}
 		else if (LinkProtocolString.Compare(TEXT("Kcp"), ESearchCase::IgnoreCase) == 0)
 		{
 			LinkProtocol = WORKER_NETWORK_CONNECTION_TYPE_MODULAR_KCP;
+			return;
 		}
-		else if (!LinkProtocolString.IsEmpty())
+
+		// None given or not recognised, default to KCP for clients, otherwise keep existing TCP default
+		if (SpatialConstants::DefaultClientWorkerType.ToString().Equals(WorkerType))
```

Comment:
```suggestion
		if (WorkerType.Equals(SpatialConstants::DefaultClientWorkerType.ToString()))
```
Ultra nit. Also think maybe == is overriden so we don't need the equal?

Reply:
The 'direction' of the comparison is to defend against if `WorkerType = nullptr`

Reply:
Hmm FString is not a pointer type, so I think you don't need to defend against nulls here, but no strong preference.
The == implementation is almost the same as equals, equals just seems to allow controlling case-sensitivity
```
	template <typename CharType>
	FORCEINLINE friend bool operator==(const FString& Lhs, const CharType* Rhs)
	{
		return FPlatformString::Stricmp(*Lhs, Rhs) == 0;
	}
```

Reply:
I'd personally make it into a switch statement for readability

Reply:
As it turns out, `WorkerType` is not set by the time this check is executed, so I am taking a different approach

Resolving this comment shortly